### ISSUE TITLE
Could not resolve module "@intlify/vue-router-bridge/lib/index.mjs" in @nuxtjs/i18n@8.0.0-rc.6

### ISSUE THREAD

**Issue by AloisSeckar** [nuxt-modules/i18n#2586]
### Environment

- Operating System: Windows_NT
- Node Version:     v18.14.0
- Nuxt Version:     3.8.2
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  npm@9.4.2
- Builder:          -
- User Config:      modules, nitro, vite
- Runtime Modules:  @nuxtjs/i18n@8.0.0-rc.6, @nuxtjs/tailwindcss@6.10.1, @pinia/nuxt@0.5.1, @vueuse/nuxt@10.6.1
- Build Modules:    -

### Reproduction

https://stackblitz.com/edit/github-jnznpl

### Describe the bug

rc.6 doesnt work with latest version of Nuxt (3.8.2)

Getting `Cannot start nuxt:  Could not resolve module "@intlify/vue-router-bridge/lib/index.mjs"`

Happens also in plain StackBlitz fork 

Reverting back to rc.5 clears the issue.

### Logs

```shell
> dev
> nuxi dev

Nuxt 3.8.2 with Nitro 2.8.1                                                                                     22:23:49
                                                                                                                22:23:50
  ➜ Local:    http://localhost:3000/
  ➜ Network:  use --host to expose


 ERROR  Cannot start nuxt:  Could not resolve module "@intlify/vue-router-bridge/lib/index.mjs"                 22:23:53  

  at setupAlias (/C:/Programming/Git/WBSC-Scoring/node_modules/@nuxtjs/i18n/dist/module.mjs:105:13)
  at async installModule (/C:/Programming/Git/WBSC-Scoring/node_modules/@nuxt/kit/dist/index.mjs:2439:95)
  at async initNuxt (/C:/Programming/Git/WBSC-Scoring/node_modules/nuxt/dist/index.mjs:3702:7)
  at async NuxtDevServer._load (/C:/Programming/Git/WBSC-Scoring/node_modules/nuxi/dist/chunks/dev2.mjs:253:5)
  at async NuxtDevServer.load (/C:/Programming/Git/WBSC-Scoring/node_modules/nuxi/dist/chunks/dev2.mjs:185:7)
  at async NuxtDevServer.init (/C:/Programming/Git/WBSC-Scoring/node_modules/nuxi/dist/chunks/dev2.mjs:180:5)
  at async Object.run (/C:/Programming/Git/WBSC-Scoring/node_modules/nuxi/dist/chunks/dev-child.mjs:101:5)
  at async runCommand$1 (/C:/Programming/Git/WBSC-Scoring/node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1648:16)
```

**Comment by alexfriesen** [comment]
I have the same problem. 

In the  function `setupAlias` some packages are correctly resolved as i will by default only lookup `node_modules` and `/node_modules/nuxt/node_modules`. But in my case some packages are installed in `node_modules/@nuxtjs/i18n/node_modules`.


A quick fix could be:
```typescript
const modulesDir = nuxt.options.modulesDir || [];
const enhancedModulesDirs = [...modulesDir, ...modulesDir.map(dir => `${dir}/@nuxtjs/i18n`)];
const module = await tryResolveModule(moduleFile, enhancedModulesDirs);
```
instead of:
```typescript
const module = await tryResolveModule(moduleFile, nuxt.options.modulesDir);
```

**Comment by andreidmour** [comment]
rc6, rc5, rc4 don't work and same error Could not resolve module "@intlify/vue-router-bridge/lib/index.mjs"

**Comment by AloisSeckar** [comment]
> rc6, rc5, rc4 don't work and same error Could not resolve module "@intlify/vue-router-bridge/lib/index.mjs"

My Nuxt project works fine with rc5. Regression happens only after trying to update to rc6.

https://github.com/AloisSeckar/WBSC-Scoring/blob/master/package.json

**Comment by BobbieGoede** [comment]
Let's continue tracking this in #2556, in the meantime I have merged @alexfriesen's PR #2587 that may resolve this issue. Please let me know if this issue has been resolved for you or not after installing the edge channel.
